<ui version="4.0" >
 <class>UsePrinterMarginsDialog</class>
 <widget class="QDialog" name="UsePrinterMarginsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Use Printer Margins</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="selectPrinterLabel" >
       <property name="text" >
        <string>Select &amp;Printer:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>printerComboBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="printerComboBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="marginsGroupBox" >
     <property name="title" >
      <string>Margins</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="4" >
         <widget class="QLineEdit" name="bottomLineEdit" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QLabel" name="rightLabel" >
          <property name="text" >
           <string>Right:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>rightLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="4" >
         <widget class="QLineEdit" name="rightLineEdit" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="leftLineEdit" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="topLabel" >
          <property name="text" >
           <string>&amp;Top:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>topLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QLabel" name="bottomLabel" >
          <property name="text" >
           <string>&amp;Bottom:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>bottomLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="leftLabel" >
          <property name="text" >
           <string>&amp;Left:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>leftLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="topLineEdit" >
          <property name="readOnly" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>212</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
